asv_bench/benchmarks/frame_methods.py

```diff
@@ -374,13 +374,13 @@ class Fillna:
 
     params = (
         [True, False],
-        ["pad", "bfill"],
+        [None, "pad", "bfill"],
         [
-            "float64",
-            "float32",
             "object",
-            "Int64",
-            "Float64",
+            "int32",
+            "int64",
```

Comment:
umm where did Int64,Float64 go?

Reply:
I replaced it with int64 and float64 so I could use the parameter as the dtype.  These parameters were just being used in the conditional flow of the setup, and I didn't see a reason for or impact from it being capitalized. 

For example:

If it was Float64 it was just avoiding all the other conditionals and was the parameter is actually never used, only:
```
values = np.random.randn(N, M)
```
And if it was Int64 it was using one extra conditional:
```
 if dtype == "Int64":
     values = values.round()
```

Reply:
pls add these back, these are on purpose as these are different types